ec/ecresource/deploymentresource/resource.go

```diff
@@ -15,24 +15,23 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package ecresource
+package deploymentresource
 
 import (
 	"time"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/terraform-providers/terraform-provider-ec/ec/ecresource/deploymentresource"
 )
 
-// Deployment returns the ec_deployment resource schema.
-func Deployment() *schema.Resource {
+// Resource returns the ec_deployment resource schema.
+func Resource() *schema.Resource {
 	return &schema.Resource{
-		CreateContext: deploymentresource.Create,
-		ReadContext:   deploymentresource.Read,
-		UpdateContext: deploymentresource.Update,
-		DeleteContext: deploymentresource.Delete,
+		CreateContext: Create,
```

Comment:
I can't leave a comment where these functions live 😄 , but it seems like they don't need to be exported anymore?